frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/DeleteBranchOnSlideOutSuggestionDialog.kt

```diff
@@ -0,0 +1,20 @@
+package com.virtuslab.gitmachete.frontend.actions.dialogs
+
+import com.intellij.ide.util.PropertiesComponent
+import com.intellij.openapi.ui.DialogWrapper
+import com.intellij.openapi.ui.Messages
+import com.virtuslab.gitmachete.frontend.actions.base.BaseSlideOutBranchAction.SLIDE_OUT_DELETION_SUGGESTION_REMEMBERED
+import com.virtuslab.gitmachete.frontend.resourcebundles.GitMacheteBundle.getString
+
+class DeleteBranchOnSlideOutSuggestionDialog : DialogWrapper.DoNotAskOption.Adapter() {
+  override fun rememberChoice(isSelected: Boolean, exitCode: Int) {
+    if (exitCode == Messages.OK && isSelected) {
+      PropertiesComponent.getInstance().setValue(SLIDE_OUT_DELETION_SUGGESTION_REMEMBERED, true)
```

Comment:
Immortal question (since my memory is poor): is it per project or per IDE? ;) Also, see the comment https://github.com/VirtusLab/git-machete-intellij-plugin/pull/656/files#r542285416 ... I think this setting can/should be fully handled by git config, to reduce the number of possible cases/combinations.

Reply:
https://jetbrains.org/intellij/sdk/docs/basics/persisting_state_of_components.html

> Use the PropertiesComponent.getInstance() method for storing application-level values, and the PropertiesComponent.getInstance(Project) method for storing project-level values.

Reply:
Ok... if that's still gonna be handled by IntelliJ properties (and not git config), then I think it's more reasonable to keep it per-project (same as with e.g. auto-adding files to git in git4idea)